### ISSUE TITLE
incorrect installation instructions?

### ISSUE THREAD

**Issue by marcelloh** [gocolly/colly#340]
go get -u github.com/gocolly/colly/...

```
go get -u github.com/gocolly/colly/...
go: finding github.com/gocolly/colly/... latest
go get github.com/gocolly/colly/...: no matching versions for query "latest"
```

**Comment by makelove** [comment]
```
localhost:~ play$ go get -u github.com/gocolly/colly/...
go get github.com/gocolly/colly/...: malformed module path "github.com/gocolly/colly/...": double dot
```
I can't install too

then I use 
go get -u github.com/gocolly/colly
OK

go get -u github.com/gocolly/colly/...
why it didn't work ?

China
macos mojave 10.14.5

**Comment by asciimoo** [comment]
Fixed in 811f2f4533d1f686fcf1a1eb516da5db3200ca60
